index.html

```diff
@@ -5,523 +5,1137 @@
-              <img src="/" alt="Headphones icon" />
-              <h4>Customer Support</h4>
-              <p>We have customer live 24/7 assistance for your needs.</p>
+              <div class="values-card-icon">
+                <svg class="icon icon-headset" width="32" height="32">
+                  <use xlink:href="../img/sprite.svg#icon-headset"></use>
+                </svg>
+              </div>
+              <div class="values-card-content">
+                <h4 class="jakarta-bold-20 card-title-overflow">
+                  Customer Support
+                </h4>
+                <p class="franklin-normal-16">
+                  We have customer live 24/7 assistance for your needs.
+                </p>
+              </div>
             </article>
           </li>
 
           <li>
             <article class="values-card">
-              <img src="/" alt="Rocket icon" />
-              <h4>Innovation Hub</h4>
-              <p>A platform for cutting-edge creative digital products.</p>
+              <div class="values-card-icon">
+                <svg class="icon icon-rocket" width="32" height="32">
+                  <use xlink:href="../img/sprite.svg#icon-rocket"></use>
+                </svg>
+              </div>
+              <div class="values-card-content">
+                <h4 class="jakarta-bold-20 card-title-overflow">
+                  Innovation Hub
+                </h4>
+                <p class="franklin-normal-16">
+                  A platform for cutting-edge creative digital products.
+                </p>
+              </div>
             </article>
           </li>
 
           <li>
             <article class="values-card">
-              <img src="/" alt="Secure sign" />
-              <h4>Secure Shopping</h4>
-              <p>We guarantee safety and privacy for every purchase process.</p>
+              <div class="values-card-icon">
+                <svg class="icon icon-shield" width="32" height="32">
+                  <use xlink:href="../img/sprite.svg#icon-shield"></use>
+                </svg>
+              </div>
+              <div class="values-card-content">
+                <h4 class="jakarta-bold-20 card-title-overflow">
+                  Secure Shopping
+                </h4>
+                <p class="franklin-normal-16">
+                  We guarantee safety and privacy for every purchase process.
+                </p>
+              </div>
             </article>
           </li>
         </ul>
       </section>
 
       <!-- Products -->
       <section class="products">
-        <h2 class="section-name">PRODUCTS</h2>
-        <h3 class="section-title">Discover Your Next Favorite Assets</h3>
+        <h2 class="section-name franklin-normal-16">PRODUCTS</h2>
+        <h3 class="section-title">
+          <span class="header-line">Discover Your</span>
+          <span class="header-line">Next Favorite Assets</span>
+        </h3>
         <figure class="products-images">
```

Comment:
Figure tag used not for group of different images 
The figure tag, in my opinion, is not entirely correctly used in this case as a wrapper for several different images. Still, it is usually used to display an image (or something else) and a caption for it

Reply:
#Fixed